packages/front-end/components/Settings/DataSourceSchemaChooser.tsx

```diff
@@ -1,45 +1,24 @@
 import { DataSourceType, SchemaFormat } from "back-end/types/datasource";
 import clsx from "clsx";
 import { FaArrowRight } from "react-icons/fa";
-
-const options: {
-  value: SchemaFormat;
-  label: string;
-  types?: DataSourceType[];
-}[] = [
-  {
-    value: "segment",
-    label: "Segment",
-  },
-  {
-    value: "ga4",
-    label: "Google Analytics v4",
-    types: ["bigquery"],
-  },
-  {
-    value: "amplitude",
-    label: "Amplitude",
-    types: ["snowflake"],
-  },
-  {
-    value: "snowplow",
-    label: "Snowplow",
-  },
-  {
-    value: "rudderstack",
-    label: "RudderStack",
-  },
-];
+import { useState } from "react";
+import Field from "../Forms/Field";
+import { MdKeyboardArrowDown, MdKeyboardArrowUp } from "react-icons/md";
+import { eventSchemas } from "../../services/eventSchema";
```

Comment:
Changes to this file are no longer being used, right?  If so, I would revert this file to keep the original DataSourceForm UI intact if needed.

Reply:
I wanted to support the same set of event trackers in either modal.